### PR TITLE
Formatting based on the standard package

### LINKED ISSUE #812: Make JavaScript comply with Standard linting
Per my discussion with @ssciolla here: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/775, I'm going to make every file conform to the linting rules of [Standard](https://standardjs.com/) and add a note about linting in the `CONTRIBUTING` doc.

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #812 
I fixed all of the files formatting that were missed or  introduced new

### CODE REVIEW

File: assets/src/util/math.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ const getDecimalPlaceOfFloat = (floatNum) => (floatNum.toString().split('.')[1])
 const average = arr => arr.reduce((acc, cur) => acc + cur, 0) / arr.length
 
 const pearsonCorrelation = (prefs, p1, p2) => {
```
Comment (pushyamig) [inline assets/src/util/math.js:10]: I don't see the function `pearsonCorrelation` being used. I just left it there, not sure what was the reason they were left off

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 52
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/my-learning-analytics/pullRequest?prid=5609374&bid=18378655)

File: .eslintrc.json [hunk 2]
```diff
@@ -11,5 +11,8 @@
 	"env": {
 		"es6": true
 	},
-	"extends": "eslint:recommended"
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/recommended"
+	]
```
Comment (pushyamig) [inline .eslintrc.json:17]: This change removed this warning 
`Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration`

**Comment by pushyamig** [comment]
@justin0022 I was wondering since the `package.json` has a lot of packages for Linting with React and we have a separate package `standard` for checking all that, Why do we need this big list
```
"eslint": "5.15.1",
    "eslint-config-react-app": "~3.0.8",
    "eslint-config-standard": "^12.0.0",
    "eslint-loader": "2.1.2",
    "eslint-plugin-flowtype": "3.4.2",
    "eslint-plugin-import": "2.16.0",
    "eslint-plugin-jsx-a11y": "6.2.1",
    "eslint-plugin-react": "7.12.4",
```

**Comment by justin0022** [comment]
Those packages are what came bundled by default with [create react app](https://github.com/facebook/create-react-app) which was used as a starting point for MyLA. I think they can be removed, but am not 100% certain.

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 52
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/my-learning-analytics/pullRequest?prid=5609374&bid=18378655)

**Comment by pushyamig** [comment]
> Those packages are what came bundled by default with create react app which was used as a starting point for MyLA. I think they can be removed, but am not 100% certain.

I have created an issue for it https://github.com/tl-its-umich-edu/my-learning-analytics/issues/947

**Comment by pushyamig** [comment]
@justin0022 if you have some time we request to review these changes will be helpful for the upcoming release in couple of weeks

**Comment by justin0022** [comment]
No problem @pushyamig I'll review now.

**Comment by justin0022** [comment]
Getting the following warning when I run `standard`: `Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .`

It's likely that when we remove eslint #https://github.com/tl-its-umich-edu/my-learning-analytics/issues/947, this warning will disappear.

File: assets/src/util/math.js (hunk 1 above)
Comment (justin0022) [reply assets/src/util/math.js:10]: I think it can be removed

File: .eslintrc.json (hunk 2 above)
Comment (pushyamig) [reply .eslintrc.json:17]: @justin0022 I have put the fix for that warning and it shouldn't be showing you that warning
I have run the `standard --fix` command and it is not showing it for me. 

Here is below

```
13:05:07-pushyami~/src/itsProjects/my-learning-analytics (Formating-issues)$ standard --fix
standard: Use JavaScript Standard Style (https://standardjs.com)
  /Users/pushyami/src/itsProjects/my-learning-analytics/sdenv/share/jupyter/nbextensions/jupyter-js-widgets/extension.js:47:354481: Parsing error: Unexpected token else
13:05:37-pushyami~/src/itsProjects/my-learning-analytics (Formating-issues)$
```
Please ignore the Jupyter-js-widgets waring it is not relevent

Comment (justin0022) [reply .eslintrc.json:17]: Ah I just realized I needed to do a docker rebuild - now that I've done so, I can confirm this warning does not display.

**Comment by justin0022** [comment]
Please feel free to merge @pushyamig

**Pull request merged**